### DISCUSSION TITLE
can't stop the process Vmmem

### DISCUSSION THREAD

**Discussion by bady0059** [microsoft/WSL#6025] [General] [answered]
I have wsl2 on my pc. Every time I open any Linux distribution I have a prosses called "vmmem" that takes 5 GB of my ram.
After I close my Linux that process keeps running. Before that I had wsl1 and it worked prefect. 

I tried to stop the process with task manager but it is not possible.
 please help.

**Comment by 0xbadfca11** [comment, marked as answer]
``wsl --shutdown`` terminate vm immediately.

vmIdleTimeout can adjust the time to terminate automatically, but it does not terminate immediately.
https://docs.microsoft.com/en-us/windows/wsl/release-notes#build-20190

**Comment by CallMePricey** [comment]
wsl.exe --shutdown doesn't always work, I believe this is because Docker or another wsl environment is still running in the background.

Close docker though task manager, next go into cmd and run  "wsl -l -v" to see what exactly is running. Note what is "Running", then run "wsl -t {Your running environment}" then run "wsl --shutdown".
